packages/backup-restore-operator/charts/README.md

```diff
@@ -0,0 +1,30 @@
+# Backup-Restore-Operator Chart
+
+The backup-restore-operator provides ability to back up and restore Kubernetes cluster metadata for user applications 
+
+### Components
+
+This chart installs the following clusterscoped CRDs  
+
+* **Backup**  
+  Creating an instance of the Backup CRD lets you perform a backup. A Backup CR must specify a ResourceSet, that contains a set of selectors to target certain resources/CRDs. (docs link TBD)
+* **Restore**  
+  Creating an instance of the Restore CRD lets you restore from a backup file. (docs link TBD)
+* **ResourceSet**  
+  ResourceSet CR should contain a set of selectors for all Kubernetes resources and CRDs that your application installs and manages, so that the operator can gather them during backups (docs link TBD)
+
+
+### User flow
```

Comment:
Also, I feel like this section in general doesn't match my perception of a User Flow; imo this section should represent step by step what a user should do to use this chart, i.e.

- First configure where you want to store your backups
- Next define a ResourceSet to select resources to be installed
- Next define a Backup CR that refers to the prior created ResourceSet so that the operator can collect and save resources. Here you can specify choosing to encrypt resources before saving them
- Once you have your operator set up to collect backups, you can trigger a restore by creating the CR that specifies the specific backup you would like to restore your cluster to

Reply:
Maybe replace this with a configuration section like in `rancher-pushprox` [here](https://github.com/rancher/charts/blob/dev-v2.5/packages/rancher-pushprox/charts/README.md#configuration)